modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -149,3 +158,33 @@
                :parameters [{:type :date/all-options
                              :target [:dimension [:template-tag "date"]]
                              :value "past30years"}]}))))))
+
+(deftest redshift-types-test
```

Comment:
you need to use `(mt/test-driver :redshift ...)` around these tests so they only run when testing a redshift driver. this is running when testing against the other drivers and breaking those tests.

Reply:
see postgres tests for an example: https://github.com/metabase/metabase/blob/master/test/metabase/driver/postgres_test.clj#L318

Reply:
Thanks; fixed that.